src/main/java/com/rhombus/api/examples/TimelapseSaver.java

```diff
@@ -0,0 +1,363 @@
+
+        if (commandLine.hasOption("vidDuration"))
+        {
+            String durationString = commandLine.getOptionValue("vidDuration");
+            duration = Integer.parseInt(durationString);
+        }
+        else
+        {
+            duration = 120;
+        }
+
+        if (commandLine.hasOption("format"))
+        {
+            format = commandLine.getOptionValue("format");
+            if(!format.equals(".mov") && !format.equals(".mp4"))
+            {
+                System.out.println("-c [.mov, .mp4]");
+                System.exit(0);
+
+            }
+        }
+        else
+        {
+            format = ".mov";
+        }
+
+        if (commandLine.hasOption("timestamp"))
+        {
+            String timestampString = commandLine.getOptionValue("timestamp");
+            if (!timestampString.equals("True"))
+            {
+                System.out.println(("[True]"));
+                System.exit(0);
+            }
+            else
+            {
+                timestamp = true;
+            }
+        }
+        else
+        {
+            timestamp = false;
+        }
+
+        if (commandLine.hasOption("cameraDetails"))
+        {
+            String cameraDetailsString = commandLine.getOptionValue("cameraDetails");
+            if (!cameraDetailsString.equals("True"))
+            {
+                System.out.println(("[True]"));
+                System.exit(0);
+            }
+            else
+            {
+                cameraDetails = true;
+            }
+        }
+        else
+        {
+            cameraDetails = false;
+        }
+
+        if (commandLine.hasOption("skipWeekends"))
+        {
+            String skipWeekendsString = commandLine.getOptionValue("skipWeekends");
+            if (!skipWeekendsString.equals("True"))
+            {
+                System.out.println(("[True]"));
+                System.exit(0);
+            }
+            else
+            {
+                skipWeekends = true;
+            }
+        }
+        else
+        {
+            skipWeekends = false;
+        }
+
+        if (commandLine.hasOption("skipNights"))
+        {
+            String skipNightsString = commandLine.getOptionValue("skipNights");
+            if (!skipNightsString.equals("True"))
+            {
+                System.out.println(("[True]"));
+                System.exit(0);
+            }
+            else
+            {
+                skipNights = true;
+            }
+        }
+        else
+        {
+            skipNights = false;
+        }
+
+        if (commandLine.hasOption("name"))
+        {
+            outputFile = commandLine.getOptionValue("name");
+        }
+        else
+        {
+            String path = System.getProperty("user.dir");
+            outputFile = path + "/timelapse" + format;;
+        }
+        String clipUuid = generateTimelapse(cameraUuid, startTime, endTime, duration, timestamp, cameraDetails, skipNights, skipWeekends, format);
```

Comment:
I would like to see some progress info being printed to the console.  Say that the timelapse is initiated, and then print the percentage complete as you keep checking the progress.  That way there is some feedback to the user that it is actually working.